extensions/amp-list/amp-list.md

```diff
@@ -312,7 +328,7 @@ We recommend using `binding="no"` or `binding="refresh"` for faster performance.
 
 If `binding` attribute is not provided, default is `always`.
 
-##### [is-layout-container](optional)
+##### is-layout-container (optional)
```

Comment:
Looks like this is a bindable attribute, so it should keep the brackets, as well as the newly added space character.

Reply:
Thanks for letting me know! I didn't know what bindable attrs get square brackets in our docs. 

In `master` right now, it lives as a broken link in the amp-list documentation. This should fix it up.

Reply:
Thanks for cleaning things up as you go!